tests/test_unit/test_graph/test_tesserae.py

```diff
@@ -1,30 +1,43 @@
 from cortexpy.tesserae import Tesserae
-
+from numpy import sqrt
 
 class TestTesserae:
-    def test_mosaic_alignment_on_short_query_and_two_templates(self):
+    def __init__(self):
         # given
-        query = "GTAGGCGAGATGACGCCAT"
-        targets = ["GTAGGCGAGTCCCGTTTATA", "CCACAGAAGATGACGCCATT"]
-
+        self.query = "GTAGGCGAGATGACGCCAT"
+        self.targets = ["GTAGGCGAGTCCCGTTTATA", "CCACAGAAGATGACGCCATT"]
         # when
-        t1 = Tesserae(mem_limit=False)
-        t2 = Tesserae(mem_limit=True)
-        p = t1.align(query, targets)
-        q = t2.align(query, targets)
+        self.t = Tesserae(mem_limit=False)
+        self.p = self.t.align(self.query, self.targets)
 
+    def test_mosaic_alignment_on_short_query_and_two_templates(self):
         # then
-        assert len(p) == 3
+        assert len(self.p) == 3
+
+        assert self.p[1][0] == 'template0'
+        assert self.p[1][1] == 'GTAGGCG'
+        assert self.p[1][2] == 0
+        assert self.p[1][3] == 6
 
-        assert p[1][0] == 'template0'
-        assert p[1][1] == 'GTAGGCG'
-        assert p[1][2] == 0
-        assert p[1][3] == 6
+        assert self.p[2][0] == 'template1'
+        assert self.p[2][1] == '       AGATGACGCCAT'
+        assert self.p[2][2] == 7
+        assert self.p[2][3] == 18
+
+        assert self.t.llk >= -27
```

Comment:
I realize this code was already here when you started working on it @karljohanw, but just out of interest, does anyone (@kvg) know what `t.llk` is and why it needs to be between -27 and -26? Also, @kvg, going forward can we please define all attributes of a class in the `__init__` section? (Ideally with documentation)